docs/roomHandlers/exampleRoom2Room.md

```diff
@@ -0,0 +1,35 @@
+# Example Room 2
```

Comment:
@jshemas This PR implements Example Room 1 which is just a room that you can leave (and do nothing else). I made example rooms 2 and 3 so that we can spec out how to handle room state. Do you want that implemented in this PR or another PR?

Reply:
They can be in a different PR.